com.bardsoftware.constable/server/src/main/kotlin/com/bardsoftware/constable/server/ConstableServer.kt

```diff
@@ -0,0 +1,41 @@
+package com.bardsoftware.constable.server
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.default
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.validate
+import com.github.ajalt.clikt.parameters.types.int
+import java.sql.DriverManager
+import java.sql.SQLException
+import java.util.logging.Level
+import java.util.logging.Logger
+
+private val logger: Logger = Logger.getLogger(ConstableServer::class.java.name)
```

Comment:
I believe that SLF4J + logback is currently the mainstream logging API in JVM world.There's also optional Kotlin wrapper for it (less boilerplate code): https://github.com/MicroUtils/kotlin-logging

Reply:
Done